_posts/2024-04-03-UsingDocumentationForUsabilityStudies.md

```diff
@@ -0,0 +1,107 @@
+
+
+### Sample considerations
+
+When you run your study, sample considerations are important. Use you persona definitions to guide sample specifications for recruitment. For larger samples and basic usability concerns, consider obtaining a paid sample. Use your screener to qualify or disqualify participants for your study. If your experience is for a niche set of users, it will make sense to recruit them by leveraging your B2B sales and services partners. For open source projects consider blogging, dropping links on GitHub, your community forum, announcing on community meetings etc. Always validate your sample with the screening questions either to 
+
+*On the “Questionnaire” page you can organize the questions for screening the participants, and also all other questions that accompany the tasks in your study:*
+![](/assets/media/blog-images/2024-04-03-UsingDocumentationForUsabilityStudies/Screenshot 2024-03-26 at 10.44.52 AM.png)
+### Reporting results
+
+Once you have taken time to conduct the studies and obtain usage data, it is time to analyze and report results. The most important aspect to report is task pass or fails. A color coded table summarizing this is very informative. See an example below. In the table, the participant qualification as expert or proficient, other tooling proficiency and the result is called out to give the reader a context on the overall findings. 
+![](/assets/media/blog-images/2024-04-03-UsingDocumentationForUsabilityStudies/image (3).png)
+In addition to tables, a good written report will include a write up of details on repetitive usage patterns, anomalies in user flows (i.e. the participant clicking around in areas that may not correct), or interpretations of icons and other design patterns. Embedding the report with user videos on points of failure or the user struggling to accomplish the task humanizes user experience research to your audience. 
+
+You can also Figma to visually depict the effectiveness and time on task across participants or fragments of the usability study report as needed:
+
+![](/assets/media/blog-images/2024-04-03-UsingDocumentationForUsabilityStudies/Screenshot 2024-03-26 at 10.45.21 AM.png)![](/assets/media/blog-images/2024-04-03-UsingDocumentationForUsabilityStudies/Screenshot 2024-03-26 at 10.45.57 AM.png)
+
+### Automation of a usability study
+
+If you have predictable user flows that you are improving the UX on etc., it may make sense to automate the study. Consider using the same usability study set up, and the same required tasks. The only thing that you would update is your end point on the back end or the link the clickable prototype. 
```

Comment:
```suggestion
If you have predictable user flows for which you are improving the UX, it may make sense to automate the study. Consider using the same study setup and the same required tasks. Only your backend endpoint or the link to the clickable prototype would need to be changed. 
```